### PR TITLE
Add DigitalTwins Tests

### PR DESCRIPTION
**Pull request by Youri970410**
## Description
Add DigitalTwins Tests

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/DigitalTwins/Changelog.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        - Additional information about change #4
+
+    ## YYYY.MM.DD - Version X.Y.Z (Previous Release)
+    * Overview of change #1
+        - Additional information about change #1
+-->
+## Upcoming Release
+
+## Version 0.1.0
+* First preview release for module Az.DigitalTwins
```
Comment (isra-fel) [inline src/DigitalTwins/Changelog.md:23]: Please do not add versions in change log. You should append new change logs right below "Upcoming release"
```suggestion
## Upcoming Release
* First preview release for module Az.DigitalTwins
```

**Review by isra-fel — COMMENTED** [review]
Could you delete `src/DigitalTwins/MSSharedLibKey.snk` file?

File: src/DigitalTwins/Az.DigitalTwins.psd1 [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+# Generated on: 2020/10/28
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = './Az.DigitalTwins.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.2.0'
```
[the remark this answers is not in the document]
Comment (Youri970410) [reply src/DigitalTwins/Az.DigitalTwins.psd1:15]: Got it,Thanks.

File: src/DigitalTwins/Changelog.md (hunk 1 above)
Comment (Youri970410) [reply src/DigitalTwins/Changelog.md:23]: Got it ,thanks.

**Review by isra-fel — DISMISSED** [review]
LGTM. Thanks!

**Review by isra-fel — APPROVED** [review]
LGTM

**Pull request merged**